### PR TITLE
Honda: add missing minimum steer speeds

### PR DESCRIPTION
**Pull request by sshane**
For autogenerating car docs: https://github.com/commaai/openpilot/pull/23762

- [ ] Do todos

CR-V:
- 2020 - Honda CR-V support and fw added: https://github.com/commaai/openpilot/pull/1464/files
- 2021 - no new eps fw added: https://github.com/commaai/openpilot/pull/20341/files

Honda Civic:

- Hatchback always has 12 mph min steer speed, but it fingerprints as sedan/coupe (`CIVIC_BOSCH`) and the fw was added for it in batches, so it's difficult to pin point the specific eps fw. Left it as a TODO.
  Diesel Bosch Civic's are always 12 mph according to the exception in CARS.md, so add simple exception for that
- To sum it up, Nidec Civic (2016-18) is 12 mph, Bosch Civic Sedan/Coupe (2019-20) is 2 mph, Bosch Civic Diesel Sedan/Coupe (2019-20) is 12 mph.

### CODE REVIEW

File: selfdrive/car/honda/interface.py [hunk 1]
```diff
@@ -218,6 +240,7 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
       ret.lateralParams.torqueBP, ret.lateralParams.torqueV = [[0, 1000], [0, 1000]]  # TODO: determine if there is a dead zone at the top end
       tire_stiffness_factor = 0.444
       ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.8], [0.24]]
+      ret.minSteerSpeed = 12. * CV.MPH_TO_MS
```
Comment (adeebshihadeh) [inline selfdrive/car/honda/interface.py:243]: if 12 is the default, no need to set explicitly here
